### ISSUE TITLE
If an @tag is @ignored then and npm run ct @tag is run, say why it's not being run

### ISSUE THREAD

**Issue by canvaspixels** [canvaspixels/courgette#18]
Currently, it blows up if you run an @ignore tag.

Say why it isn't running and stop it blowing up

**Comment by canvaspixels** [comment]
Fixed in 2.17.0

**Comment by canvaspixels** [comment]
Just add @ignore tags at the feature level or scenario level as shown below:

```
@simplepage @ignore
Feature: simple page feature

  Background: I am on the simple page
    Given I am on the 'simple' page

  @simplepage-react-link @ignore
  Scenario: Testing .page file for css selectors
    When I click the 'Go to home page by react router link'
    Then I expect to be on the 'home' page
    Then I expect the 'main container' to be visible
    And take a screenshot called 'home page'
```
